_posts/2018-05-16-inception-pipelines-pt3.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Seeds of Inception - Part 3"
+date:   2018-05-16
+categories: aws continuous deployment cdn SPA cloudfront cross-account
+author: Pete Yandell
+image: img/inception-pipelines/seed_germination.png
+---
+
+**Part 3 sharing the website goodness** - A practical example for DevOps on AWS
+
+<a title="By Dbxsoul [CC BY 3.0 (https://creativecommons.org/licenses/by/3.0)], from Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File:Germinating_seedling.jpg"><img width="512" alt="Germinating seedling" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/60/Germinating_seedling.jpg/512px-Germinating_seedling.jpg"></a>
+
+## What's The Problem
+
+Welcome back to the 3rd instalment in the "Seeds of Inception" series, where we build upon [Part 2](https://mechanicalrock.github.io//aws/continuous/deployment/cdn/spa/cloudfront/2018/04/01/inception-pipelines-pt2) by setting our pipelines to 'defrost'. By this I mean solving the snowflake environment issue that plagues software development; where each environment is unique and special and just a little bit different from all the others in the DEV to PROD chain.
+
+Getting down to the guts of it, I'll demonstrate one way to deploy your single `aws_infrastructure.yml` template across multiple AWS Accounts. All from the comfort of your very own [Inception Pipeline](https://github.com/MechanicalRock/InceptionPipeline/tree/post/part-3)
```

Comment:
Might want to use this https://aws.amazon.com/blogs/devops/construct-your-own-launch-stack-url/ so that people can test this without needing to create github account and checkout code